tests/app/dao/test_service_sms_sender_dao.py

```diff
@@ -155,6 +155,24 @@ def test_dao_add_sms_sender_for_service(self, notify_db_session, sample_provider
 
         assert new_sms_sender in service_sms_senders_after_updates
 
+    def test_dao_add_sms_sender_256_char(self, notify_db_session, sample_provider, sample_service) -> None:
+        provider = sample_provider()
+        service = sample_service()
+
+        new_sms_sender = dao_add_sms_sender_for_service(
```

Comment:
Doesn't this require tear down?

Reply:
Fortunately it does not. Could potentially use a comment that `service_cleanup` will take care of it though.

Reply:
Fortunately it does not. Could potentially use a comment that service_cleanup will take care of it though.

Reply:
added a comment about `service_cleanup`